src/applications/edu-benefits/10282/containers/ConfirmationPage.jsx

```diff
@@ -52,13 +52,42 @@ export class ConfirmationPage extends React.Component {
               <span>{format(submitDate, 'MMMM d, yyyy')}</span>
             </p>
           ) : null}
+          <p>
+            <strong>Confirmation for your records </strong>
+            <br />
+            <span>
+              You can print this confirmation page for your records. You can
+              also download your completed application as a PDF.
+            </span>
+          </p>
+          {/* eslint-disable-next-line @department-of-veterans-affairs/prefer-button-component */}
           <button
             type="button"
             className="usa-button screen-only"
             onClick={window.print}
           >
-            Print this for your records
+            Print this page
           </button>
+          <va-link
+            download
+            class="vads-u-display--block"
+            filetype="PDF"
+            href="/education/apply-for-education-benefits/10282/confirmation"
+            text="Download your completed application"
+          />
+        </div>
+        <div className="vads-u-margin-top--0">
+          <h2>What to expect next</h2>
+          <p>
+            If you're accepted into the SkillsBuild program, you'll receive an
+            email from souser@us.ibm.com with your login informatia. If you
```

Comment:
Should the email be an achor tag with a mailto?

Reply:
done